pkg/source/client/client.go

```diff
@@ -0,0 +1,64 @@
+package client
```

Comment:
Use a different package other than `pkg/source/xxx`, that should be left to real user defined source.